riscv_config/constants.py

```diff
@@ -6,19 +6,34 @@
 isa_schema = os.path.join(root, 'schemas/schema_isa.yaml')
 debug_schema = os.path.join(root, 'schemas/schema_debug.yaml')
 platform_schema = os.path.join(root, 'schemas/schema_platform.yaml')
+Zvl_extensions = [
+        "Zvl32b", "Zvl64b", "Zvl128b", "Zvl256b", "Zve512b", "Zvl1024b"
+]
+Zvef_extensions = [
+        "Zve32f", "Zve64f"
+]
+Zved_extensions = [
+        "Zve64d"
+]
+Zve_extensions = [
```

Comment:
I doubt that this ordering is alphabetical. However, there is ambiguity on how to order the extensions when there are numbers included in the names. However, the `zved` extensions should come before `zvef`.

Reply:
The order of these sub-extensions should be irrelevant since they can only appear once in the ISA string.

Reply:
The ordering is especially important because it determines the position of the extension in the string when multiple of them are present. That being said, I think the spec is very ambiguous on the ordering of `Z` extensions when numbers are involved. This will need to be fixed at a later point.

Reply:
IIUC, `Zve*`/`Zvl*` can appear at most once in a valid ISA string, so their order in the array doesn't matter, right?

Reply:
Do you mean that the extensions are mutually exclusive i.e `Zve32d` and `Zve32f` cannot appear together?

Reply:
Yes, sorry for my bad English.